exampleCourse/questions/demo/randomFakeData/server.py

```diff
@@ -23,7 +23,7 @@ def generate(data):
     for alias, birthday in zip(
```

Comment:
Approved the other one too fast, looking at the above code, realizing that the birthday uniqueness can be done with the same call instead of creating a set.

Reply:
Done in fd6e1a6.